template/client/components/ActionBar.vue

```diff
@@ -11,24 +11,24 @@ import type { CommentType } from '~/types'
 // @vue/component
 @Component()
 /**
-* Actions bar component written in jsx.
-* The main idea of this example is to demostrate flexibility of Vue and jsx.
-* We discourage using `jsx` in a real world apps,
-* unless you know what are you doing!
-*/
+ * Actions bar component written in jsx.
```

Comment:
Could you please revert `jsdoc` spacing? We stick to

```
/**
*
*/
```

That's strange that linting actually allows that. I have created a new issue.
https://github.com/wemake-services/wemake-vue-template/issues/493